figure2_plot.py

```diff
@@ -90,7 +100,10 @@ def plot_pareto_frontier(Xs, Ys, color, linestyle, maxX=True, maxY=True):
         for fn in args.data_filename:
             with open(fn) as f:
                 for l in f:
-                    proto, util, delay, throughput, power = l.split(', ')
+                    proto, util, delay, throughput, power, queuing_delay = l.split(', ')
+                    if args.use_queuing_delay:
```

Comment:
Didn't we nix this? I guess it doesn't really matter if we leave it.

Reply:
Yeah, it really doesn't affect anything.  I think we can just keep it in case we need to use this for whatever reason down the line.